blueprints/addon/files/README.md

```diff
@@ -3,6 +3,14 @@
 
 [Short description of the addon.]
 
+
+Compatibility
+------------------------------------------------------------------------------
+
+* Ember.js v2.18 or above
+* Ember CLI 2.13 or above
```

Comment:
@kennethlarsen sorry, I missed it too above, but could you change it to `v2.13` so that it matches the Ember.js line formatting?